scout/server/blueprints/institutes/controllers.py

```diff
@@ -279,18 +279,16 @@ def get_sanger_unevaluated(store, institute_id, user_id):
     return unevaluated
 
 
-def gene_variants(store, variants_query, institute_id, page=1, per_page=50):
+def gene_variants(store, pymongo_cursor, institute_id, page=1, per_page=50):
```

Comment:
I would prefer type hints so we get both information that it is a cursor and that it includes variants, so something like:
```suggestion
def gene_variants(store: Adapter, variants: pymongo.Cursor, institute_id: str, page: int = 1, per_page: int = 50):
```

Reply:
This would be a separate refactoring pull request.